doc/api/cli.md

```diff
@@ -2017,7 +2017,7 @@ changes:
 > Stability: 1.2 - Release candidate
 
 This runs a specified command from a package.json's `"scripts"` object.
-If no `"command"` is provided, it will list the available scripts.
+If an invalid `"command"` is provided, it will list the available scripts.
```

Comment:
```suggestion
If a missing `"command"` is provided, it will list the available scripts.
```

Reply:
I think "invalid" is better. `node --run` is still missing a command so it is not different from the original wording.

Reply:
How about "If there is not a matching "command", it will list the available scripts."?